scripts/release.sh

```diff
@@ -16,4 +16,31 @@ CARGO_DEPS=control-plane.project-builder.cargoDeps
 PROJECT="controller"
 . "$SOURCE_REL"
 
+upload_images() {
```

Comment:
this should just be a submodule upgrade not a code change here?

Reply:
Fixed